libqtile/widget/sensors.py

```diff
@@ -81,9 +81,14 @@ def get_temp_sensors(self):
         temperature_list = {}
         temps = psutil.sensors_temperatures(fahrenheit=not self.metric)
         unit = '°C' if self.metric else '°F'
+        empty_index = 0
         for kernel_module in temps:
             for sensor in temps[kernel_module]:
-                temperature_list[sensor.label] = (str(round(sensor.current, 1)), unit)
+                label = sensor.label
+                if not label:
+                    label = 'UNKNOWN-' + str(empty_index)
```

Comment:
Could you use `kernel_module` rather than `UNKNOWN` (but fallback to `UNKNOWN` if `kernel_module` is also blank)?

Reply:
Sure thing.